test/system/repository.test.js

```diff
@@ -46,24 +44,24 @@ describe('project repository', function () {
             it('should have valid, existing files', function () {
                 var scriptRegex = /^node\snpm\/.+\.js$/;
 
-                expect(json.scripts).to.be.ok();
+                expect(json.scripts).to.be.ok;
                 json.scripts && Object.keys(json.scripts).forEach(function (scriptName) {
-                    expect(scriptRegex.test(json.scripts[scriptName])).to.be.ok();
-                    expect(fs.statSync('npm/' + scriptName + '.js')).to.be.ok();
+                    expect(scriptRegex.test(json.scripts[scriptName])).to.be.ok;
+                    expect(fs.statSync('npm/' + scriptName + '.js')).to.be.ok;
                 });
             });
 
             it('should have the hashbang defined', function () {
                 json.scripts && Object.keys(json.scripts).forEach(function (scriptName) {
                     var fileContent = fs.readFileSync('npm/' + scriptName + '.js').toString();
-                    expect(/^#!\/(bin\/bash|usr\/bin\/env\snode)[\r\n][\W\w]*$/g.test(fileContent)).to.be.ok();
+                    expect(/^#!\/(bin\/bash|usr\/bin\/env\snode)[\r\n][\W\w]*$/g.test(fileContent)).to.be.ok;
```

Comment:
`expect(fileContent).to.match(/^#!\/(bin\/bash|usr\/bin\/env\snode)[\r\n][\W\w]*$/g);`